### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by leila-cit**

### CODE REVIEW

**Review by diegok-ciandt — APPROVED** [review]

**Review by diegok-ciandt — APPROVED** [review]

File: app/src/main/java/com/example/leila/exercise1/ListActivity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class ListActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_list);
+        //getActionBar().setDisplayHomeAsUpEnabled(true);
```
Comment (glira-ciandt) [inline app/src/main/java/com/example/leila/exercise1/ListActivity.java]: Can this commented code be removed?

File: app/src/main/java/com/example/leila/exercise1/MainActivity.java [hunk 2]
```diff
@@ -29,7 +30,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @OnClick(R.id.btnList)
     public void OnBtnListClicked(View view){
-        Toast.makeText(MainActivity.this, "Botão 1 clicado", Toast.LENGTH_SHORT).show();
+        Intent i = new Intent(getApplicationContext(),ListActivity.class);
```
Comment (glira-ciandt) [inline app/src/main/java/com/example/leila/exercise1/MainActivity.java]: Please, add a small space before ListActivity.class

File: app/src/main/java/com/example/leila/exercise1/SplashActivity.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+public class SplashActivity extends AppCompatActivity implements CountdownListener{
+
+    private static final int SPLASH_DISPLAY_TIME = 4000;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        new Timer(SPLASH_DISPLAY_TIME,1000,this).start();
```
Comment (glira-ciandt) [inline app/src/main/java/com/example/leila/exercise1/SplashActivity.java]: Please, add some spaces between the arguments.

File: app/src/main/java/com/example/leila/exercise1/Timer.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+     */
+    public Timer(long millisInFuture, long countDownInterval, CountdownListener listener) {
+        super(millisInFuture, countDownInterval);
+
+        this.listener = listener;
+    }
+
+    @Override
+    public void onTick(long millisUntilFinished) {
+
```
Comment (glira-ciandt) [inline app/src/main/java/com/example/leila/exercise1/Timer.java]: Please, add a //Not implemented comment in this method

**Review by glira-ciandt — CHANGES_REQUESTED** [review]

**Review by glira-ciandt — APPROVED** [review]

**Pull request merged**